javascript/javascript1/week3/seriesTime.js

```diff
@@ -0,0 +1,24 @@
+const calculateTime = [
+    {
+      title: 'a',
+      days: 9, 
+      hours: 22,
+      minutes: 32,
+    },
+    {
+      title: 'b',
+      days: 3,
+      hours: 14,
+      minutes: 0,
+    },
+    {
+      title: 'c',
+      days: 6,
+      hours: 8,
+      minutes: 0,
+    }
+  ]
+  for (let {title, days, hours, minutes} of calculateTime) {
+    let fraction = ((days*24 + hours)*60 + minutes) / (365*24*60);
```

Comment:
365*24*60 is for one year, you're asked about your whole life

Reply:
https://github.com/Aswini-D3/hyf-homework/blob/javascript/javascript1/week3/javascript/javascript1/week3/seriesTime.js

I changed the code for 80 years, can you please cehck once.